mlflow/entities/assessment.py

```diff
@@ -177,7 +177,10 @@ def to_dictionary(self):
 @experimental
 @dataclass
 class Expectation(_MlflowObject):
-    """Represents an expectation value in an assessment."""
```

Comment:
The current description doesn't tell users what this object is used for / intended to accomplish. I updated the docstring in an attempt to be more descriptive.